x-pack/plugins/security/server/lib/authorization/features_privileges_builders.test.ts

```diff
@@ -92,13 +96,46 @@ describe('#buildFeaturesPrivileges', () => {
     });
   });
 
-  test('includes app actions when specified', () => {
+  test('includes app actions when specified at the feature level', () => {
     const actions = new Actions(versionNumber);
     const builder = new FeaturesPrivilegesBuilder(actions);
     const features = [
       {
         id: 'foo',
         name: '',
+        app: ['foo-app', 'bar-app'],
+        privileges: {
+          bar: {
+            savedObject: {
+              all: [],
+              read: [],
+            },
+            ui: [],
+          },
+        },
+      },
+    ];
+    const result = builder.buildFeaturesPrivileges(features);
+    expect(result).toEqual({
+      foo: {
+        bar: [
+          actions.login,
+          actions.version,
+          actions.app.get('foo-app'),
+          actions.app.get('bar-app'),
+        ],
+      },
+    });
+  });
+
+  test('includes app actions when specified at the feature and privilege level', () => {
```

Comment:
We have the same apps specified at the feature and privilege level, what exactly are we testing here?

Reply:
hmm. Not sure what I was thinking when I put this together...